apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog.tsx

```diff
@@ -118,6 +122,7 @@ function CustomEventDialogs(props: CustomEventDialogProps) {
             end: end,
             customEventID: props.customEvent ? props.customEvent.customEventID : Date.now(),
             building: building,
+            term: getLatestTermByShortName(termsShortNames)?.shortName,
```

Comment:
What does this do? If it gets the latest term of all sections in the schedule, this should be assigned to an intermediate variable (something like `latestTermInSchedules`).